src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -192,10 +192,10 @@ public FeedValidationResultSummary validationSummary() {
     public String namespace;
 
     /**
-     * Indicates whether a feed version is pending published status, a check that is currently performed in
-     * {@link com.conveyal.datatools.manager.jobs.FeedUpdater}. This field is currently in use only for the MTC extension.
+     * Indicates when (if at all) a feed version was published to an external source. This field is currently in use
+     * only for the MTC extension and is reset to null after the published version has been registered externally.
      * */
-    public boolean processing;
+    public Date published;
```

Comment:
Can we rename this variable to `externallyPublished`?  Also, I don't quite understand why it's being set to null.

Reply:
@evansiroky, yes we can change that variable name. As for why it's being set to null, there are three states a version can be for the UI: published/processed by the external source, which is indicated by `FeedSource#publishedVersionId`; processing, which is indicated by a non-null value in this field (`FeedVersion#externallyPublished`); and not published, which is indicated by a null value in this field.

I figured using null values here was better than leaving the date value intact and requiring the UI to compare these values to find the most recent (which would indicate which version is currently processing). I do realize though that this is a bit of a strange pattern. Do you have any thoughts on improving it?

Reply:
One idea for improvement would be to change the variable name to something like `timeSentToExternalPublisher` then create another variable of something like `timeWhenExternalPublisherAppearedToHaveFinishedProcessingFeedVersion`.  I don't think overwriting the variable to null is a good idea.  First, it erases information.  Second, when the FeedVersion `published` variable is null, then that has meaning attached to it.  I think it would be a lot better to set another variable to a non-null state to convey a positive state.